serverless/serverless-getting-started.adoc

```diff
@@ -8,6 +8,8 @@ toc::[]
 
 {ServerlessProductName} simplifies the process of delivering code from development into production by reducing the need for infrastructure set up or back-end development by developers.
 
+Serverless is a cloud computing model where application developers don't need to provision servers or manage scaling for their applications. These routine tasks are abstracted away by the platform, allowing developers to push code to production much faster than in traditional models.
```

Comment:
s/don't/do not

Reply:
s/much faster/more quickly ??

_must faster_ doesn't sound right to me, jargon-y?  Up to you.